lib/gutentag/tag_names.rb

```diff
@@ -1,21 +0,0 @@
-class Gutentag::TagNames
-  def self.call(*names)
-    new(names).call
-  end
-
-  def initialize(names)
-    @names = names.flatten
-  end
-
-  def call
-    name_values.collect { |name| Gutentag::TagName.call name }
```

Comment:
I was thinking TagNames was used in multiple places, hence making it re-usable. But you're right, it's not, so I guess we can lose it 👍